cogs/mod.py

```diff
@@ -49,9 +51,10 @@ def __init__(self, bot):
         dest = user
         await dest.send(
             content='{}\n*To answer write* `{}answer "your message here"`'.
-            format(message, self.bot.config.command_prefix))
-        channel_to_forward = self.bot.get_channel(
-            self.bot.config.reception_channel_id)
+            format(message, self.bot.config.command_prefix[0]))
```

Comment:
ok as an aside why is this an array in the first place? can there be more than one?

Reply:
it seems so, but also when you send multiple pictures it always sends them in different messages so i'm not sure when this happens

Reply:
the iOS app allows multiple pics per message. not sure about android. definitely not an option on desktop.